### ISSUE TITLE
Suggested revisions to the Associations documentation

### ISSUE THREAD

**Issue by stscijgbot-jp** [spacetelescope/jwst#5814]
_Issue [JP-1951](https://jira.stsci.edu/browse/JP-1951) was created on JIRA by [Rosa Diaz](https://jira.stsci.edu/secure/ViewProfile.jspa?name=rdiaz):_

I finished reviewing the JWST Associations section of the Associations documentation. This does not mean I am done as I am still looking at where and how to add the actual rules for all the observing modes; for this, I am looking at the Associations code (more later). Regardless, I think all the comments here apply; except for probably the last part where the actual rules might have to be introduced.

Let me know if I can help with these changes.

Also attempt to resolve [github #&#x2060;2556](https://github.com/spacetelescope/jwst/issues/2556).

**Comment by stscijgbot-jp** [comment]
_Comment by [Jonathan Eisenhamer](https://jira.stsci.edu/secure/ViewProfile.jspa?name=eisenham) on [JIRA](https://jira.stsci.edu/browse/JP-1951?focusedCommentId=533792#comment-533792):_

[Rosa Diaz](https://jira.stsci.edu/secure/ViewProfile.jspa?name=rdiaz) I've done nearly all the updates that were suggested in the original attached document. The work is in [PR #&#x2060;6156](https://github.com/spacetelescope/jwst/pull/6156).

A slightly modified document has been uploaded with the date 6-17-2021. All the issues resolved in the PR has been marked with a green check.

The only issues that have not been done are the rule-specific documentation. These pages clearly need much work and will leave to near-future work.

If you can review what has been done to ensure I'm on the right track, that would be great.

**Comment by stscijgbot-jp** [comment]
_Comment by [Rosa Diaz](https://jira.stsci.edu/secure/ViewProfile.jspa?name=rdiaz) on [JIRA](https://jira.stsci.edu/browse/JP-1951?focusedCommentId=534440#comment-534440):_

I have made my review of the changes. Regarding the Level 3 association rules, we have mentioned that this information probably should live in JDox. I started working on this but got new ideas on how to approach the problem, so started investigating that. Will chat with Jonathan later about this.

**Comment by stscijgbot-jp** [comment]
_Comment by [Howard Bushouse](https://jira.stsci.edu/secure/ViewProfile.jspa?name=bushouse) on [JIRA](https://jira.stsci.edu/browse/JP-1951?focusedCommentId=535283#comment-535283):_

Fixed by #6156

**Comment by stscijgbot-jp** [comment]
_Comment by [Jonathan Eisenhamer](https://jira.stsci.edu/secure/ViewProfile.jspa?name=eisenham) on [JIRA](https://jira.stsci.edu/browse/JP-1951?focusedCommentId=533792#comment-533792):_

[Rosa Diaz](https://jira.stsci.edu/secure/ViewProfile.jspa?name=rdiaz) I've done nearly all the updates that were suggested in the original attached document. The work is in [PR #&#x2060;6156](https://github.com/spacetelescope/jwst/pull/6156).

A slightly modified document has been uploaded with the date 6-17-2021. All the issues resolved in the PR has been marked with a green check.

The only issues that have not been done are the rule-specific documentation. These pages clearly need much work and will leave to near-future work.

If you can review what has been done to ensure I'm on the right track, that would be great.

**Comment by stscijgbot-jp** [comment]
_Comment by [Rosa Diaz](https://jira.stsci.edu/secure/ViewProfile.jspa?name=rdiaz) on [JIRA](https://jira.stsci.edu/browse/JP-1951?focusedCommentId=534440#comment-534440):_

I have made my review of the changes. Regarding the Level 3 association rules, we have mentioned that this information probably should live in JDox. I started working on this but got new ideas on how to approach the problem, so started investigating that. Will chat with Jonathan later about this.
